elisp-bytecode.texi

```diff
@@ -59,22 +59,35 @@ A stack reference
 @node byte-varref
 @unnumberedsubsec @code{byte-var-ref} (8--15)
 @kindex byte-varref
-A variable reference.
+Loads the value of a variable reference on the the evaluation stack.
 
 @subsubsection Example
 
 @code{(defun en(n) n)} generates:
 @verbatim
 PC byte Instruction
-0   8 (byte-varref n)
-1 135 (byte-return . 0)
+0     8 (byte-varref n)  ;; loads variable n onto the stack
+1   135 (byte-return . 0)
 @end verbatim
 
 @node byte-varset
 @unnumberedsubsec @code{byte-varset} (16--23)
 @kindex byte-varset
 
-Sets a variable
+Sets a variable given in the operand to the value that is on the top
+of the stack.
+
+@subsubsection Example
+
+@code{(defun en(n5) (setq n 5)} generates:
```

Comment:
I'm just noticing because I'm putting together a function to re-generate the examples, but we're missing a `)` here.

Reply:
Also, the `n5` argument is unused, which is a little confusing.

Reply:
Ok. Both of these are in the next pull request.